emscripten/CMakeLists.txt

```diff
@@ -2,7 +2,15 @@ cmake_minimum_required(VERSION 3.0.0)
 
 # ------------------------------------------------------------------------------
 
-SET(emscripten_root_path "$ENV{EMSCRIPTEN}")
+
+if(DEFINED ENV{EMSCRIPTEN})
+    SET(emscripten_root_path "$ENV{EMSCRIPTEN}")
+else()
+    find_file(EMCC_EXECUTABLE emcc HINTS "/usr/lib/emscripten")
```

Comment:
As far as I understand `CMAKE_INSTALL_LIBDIR` it is defined as either _lib_ or _lib64_. If that is the case then no, emscripten so far is 32bit only.

Reply:
Okay... that's what I was wondering, whether it could be in e.g. `/usr/lib64`. If it is always in `/usr/lib` even on platforms that have a `/usr/lib64`, then this is fine. (For that matter, even if it's wrong, it's just an inconvenience.)